python/cudf/cudf/core/column/string.py

```diff
@@ -5629,7 +5630,7 @@ def __init__(
         null_count: int | None = None,
         children: tuple["column.ColumnBase", ...] = (),
     ):
-        dtype = cudf.api.types.dtype("object")
+        dtype = np.dtype("object")
```

Comment:
Should we use CUDF_STRING_DTYPE here? I'm not sure how you envisioned those constants being used.